codalab/lib/bundle_cli.py

```diff
@@ -88,13 +88,15 @@
 from codalabworker.file_util import remove_path
 
 # Formatting Constants
-GLOBAL_SPEC_FORMAT = "[<alias>::|<address>::](<uuid>|<name>)"
 ADDRESS_SPEC_FORMAT = "(<alias>|<address>)"
-TARGET_SPEC_FORMAT = '(<uuid>|<name>)[%s<subpath within bundle>]' % (os.sep,)
-ALIASED_TARGET_SPEC_FORMAT = '[<key>:]' + TARGET_SPEC_FORMAT
+BASIC_SPEC_FORMAT = '(<uuid>|<name>)'
 BUNDLE_SPEC_FORMAT = '(<uuid>|<name>|^<index>)'
-GLOBAL_BUNDLE_SPEC_FORMAT = '((<uuid>|<name>|^<index>)|(<alias>|<address>)::(<uuid>|<name>))'
+GLOBAL_SPEC_FORMAT = "[%s::]%s" % (ADDRESS_SPEC_FORMAT, BASIC_SPEC_FORMAT)
 WORKSHEET_SPEC_FORMAT = GLOBAL_SPEC_FORMAT
+REMOTE_BUNDLE_SPEC_FORMAT = '(%s//%s)' % (WORKSHEET_SPEC_FORMAT, BUNDLE_SPEC_FORMAT)
```

Comment:
Don't use the word "remote" - that suggests a different CodaLab instance - e.g., "main::foo".  I might even swap 'remote' and 'global', but probably there's something better.